src/platform_impl/macos/window_delegate.rs

```diff
@@ -332,6 +380,18 @@ declare_class!(
             if let Some(target_fullscreen) = target_fullscreen {
                 self.window.set_fullscreen(target_fullscreen);
             }
+
+            if let Some(screen) = self.window.screen() {
+                let rect = screen.frame();
+                let logical_size =
+                    LogicalSize::new(rect.size.width as f64, rect.size.height as f64);
+                let size = logical_size.to_physical::<u32>(self.window.scale_factor());
+
+                self.queue_event(WindowEvent::Configured {
+                    size,
+                    state: Default::default(),
+                });
+            }
```

Comment:
In general, won't it all trigger the frame changes or other events? Can't we simply get the all the stiles of `zoomed`, `fullscreen` inside the resize handle or macOS won't resize to the same sizes? Do we have, maybe, some state changes hooks?

Reply:
We should use the delegate methods now they are there, imo. That's cleanest imo.